src/plays/model/PlayUpdateRequest.ts

```diff
@@ -0,0 +1,6 @@
+import Play from './Play';
+
+export default interface PlayUpdateRequest extends Partial<Play> {
+  createdAt?: Date;
```

Comment:
Why do you need the createdAt and updatedAt in this request? Can you please remove them